ecr-login/cli/docker-credential-ecr-login/main.go

```diff
@@ -14,13 +14,36 @@
 package main
 
 import (
+	"flag"
+	"fmt"
+	"os"
+
 	ecr "github.com/awslabs/amazon-ecr-credential-helper/ecr-login"
 	"github.com/awslabs/amazon-ecr-credential-helper/ecr-login/api"
 	"github.com/awslabs/amazon-ecr-credential-helper/ecr-login/config"
+	"github.com/awslabs/amazon-ecr-credential-helper/ecr-login/version"
 	log "github.com/cihub/seelog"
 	"github.com/docker/docker-credential-helpers/credentials"
 )
 
+var vrsn bool
```

Comment:
Can you rename this to `versionFlag`?  Also, can you move it inside `main()`?

Reply:
Fixed by 527b0da